### ISSUE TITLE
IndieWeb forward thinking comments.php output

### ISSUE THREAD

**Issue by chrisaldrich** [pfefferle/SemPress#39]
Given that this theme is one of the exceedingly few that supports microformats 2 and that it stands as the (only?) definitive theme in the IndieWeb echosystem for WordPress, might I suggest that updating the "typical" [comments.php](https://github.com/pfefferle/SemPress/blob/master/sempress/comments.php) be modified to better reflect integration for use with webmentions could help catapult uptake of the idea of the indieweb movement? By this I mean customizing the "typical" WordPress comments.php output to sort webmentions, pingbacks, and "traditional" comments to display in something like the following suggested order:

1. a facepile of "likes" (potentially even separated by silos)
2. a facepile of "reposts" (potentially separated by supported silo)
3. the traditional list of comments
4. the list of pingbacks/trackbacks (potentially showing the actual content rather than the usual "Joe mentioned this post" output)

but doing so in a way that doesn't cause issues (potentially?) for those not living in the IndieWeb echosystem.  From a UI perspective something like this is far superior to scrolling through possibly hundreds of "likes" and "repost" comments to get to more substantive discussions in comments.

I suspect that @dshanske is attempting to modularize this type of functionality in a plugin with [semantic comments](https://github.com/dshanske/semantic-comments) (which doesn't seem to work at the moment), but having at least one example theme with a straightforward version of a better handling of comments (also including microformats) might assist in the uptake of theme developers doing a more modern version of WordPress comments using webmention. (And naturally even better if it can be done canonically enough that WordPress subsumes it into core thereby infecting a multitude of sites to be IndieWeb and microformat compatible in one go.) I might also suggest that standardization in core results in broader and quicker uptake than the need to find and install a plugin.

As an example, I'll note that the [Independent Publisher theme](https://github.com/raamdev/independent-publisher/) has shown interest in being more IndieWeb compatible, but some of the custom code for comments is preventing showing likes and reposts (see https://github.com/raamdev/independent-publisher/issues/201).

**Comment by dshanske** [comment]
I had tried that. The last rewrite I did of the comment improvements was inside the mf2_s repo(also something I should get back to someday).

https://github.com/dshanske/mf2_s/blob/master/inc/class-comment-walker.php

I'd like to roll a version of this into Semantic Linkbacks at some point.

**Comment by jihaisse** [comment]
Something like that : http://jihais.se/2016/03/12/petite-soiree-entre-amis/ ?

**Comment by chrisaldrich** [comment]
Yes, @jihaisse that's a great example (though it doesn't have present examples of reposts or pingbacks/trackbacks at the moment.)
